controller/src/main/java/io/pravega/controller/server/rest/resources/StreamMetadataResourceImpl.java

```diff
@@ -320,6 +320,45 @@ public void deleteStream(final String scopeName, final String streamName, final
         .thenAccept(x -> LoggerHelpers.traceLeave(log, "deleteStream", traceId));
     }
 
+    @Override
+    public void deleteReaderGroup(final String scopeName, final String readerGroupName,
+                                  final SecurityContext securityContext, final AsyncResponse asyncResponse) {
+        long traceId = LoggerHelpers.traceEnter(log, "deleteReaderGroup");
+        long requestId =  requestIdGenerator.nextLong();
+
+        try {
+            restAuthHelper.authenticateAuthorize(
+                    getAuthorizationHeader(),
+                    authorizationResource.ofReaderGroupInScope(scopeName, readerGroupName),
+                    READ_UPDATE);
+        } catch (AuthException e) {
+            log.warn(requestId, "delete reader group for {} failed due to authentication failure.",
```

Comment:
may be -
`log.warn(requestId, "delete reader group for scope: {}, and rederGroup: {} failed due to authentication failure.",scopeName, readerGroupName)`